<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QRWirelessConnect - WiFi QR Generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrc.qrc">
    <normaloff>:/img/logo.png</normaloff>:/img/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="1">
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="10" column="2">
        <widget class="QLabel" name="label_strength">
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:red;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>SSID:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="10" column="0" rowspan="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Strength:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="3">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 12pt ;</string>
         </property>
         <property name="text">
          <string>Wifi Passphrase:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEdit_SSID"/>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEdit_Pass_1">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Auth:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_Pass_1">
         <property name="text">
          <string>Passphrase:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_Pass_2">
         <property name="text">
          <string>Confirm:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEdit_Pass_2">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox_auth">
         <item>
          <property name="text">
           <string>WPA2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WPA3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WPA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WEP</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>370</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: gray;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_qr">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>350</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>674</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy_QR_to_Clipboard"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionE_xit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset theme="printer">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Print...</string>
   </property>
  </action>
  <action name="actionCopy_QR_to_Clipboard">
   <property name="text">
    <string>Copy QR to Clipboard</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEdit_SSID</tabstop>
  <tabstop>lineEdit_Pass_1</tabstop>
  <tabstop>lineEdit_Pass_2</tabstop>
  <tabstop>comboBox_auth</tabstop>
 </tabstops>
 <resources>
  <include location="../qrc.qrc"/>
 </resources>
 <connections/>
</ui>
